Livecheckables/clib.rb

```diff
@@ -0,0 +1,6 @@
+class Clib
+  livecheck do
+    url :stable
```

Comment:
```suggestion
    url :head
```

The formula provides the GitHub repo as the `head` URL, so we can simply use that instead of `:stable` + URL preprocessing.